lib/info.js

```diff
@@ -67,6 +67,14 @@ exports.getBasicInfo = (id, options, callback) => {
       related_videos: extras.getRelatedVideos(body),
     };
 
+    // Get likes
+    const likes = extras.getLikes(body);
+    if (likes) additional.likes = likes;
+    
+    // Get dislikes
+    const dislikes = extras.getDislikes(body);
+    if (dislikes) additional.dislikes = dislikes;
```

Comment:
can you put these above with the rest of the `additional` properties? it's ok if they're `null`